RUFAS/routines/animal/ration/ration_driver.py

```diff
@@ -1003,10 +994,11 @@ def get_forage_NDF(
             Forage neutral detergent fiber content of feed item, g.
 
         """
+        forage_NDF_item = 0.0
+
         if feed_item_info["feed_type"] == "Forage":
             forage_NDF_item = feed_item_info["NDF"] * kg_fed
-        else:
-            forage_NDF_item = 0.0
+
         return forage_NDF_item
```

Comment:
```suggestion
        return feed_item_info["NDF"] * kg_fed if feed_item_info["feed_type"] == "Forage" else 0
```
Since the logic here is so simple, what do you think of using the short notation?

Reply:
Good improvement!